vendor/github.com/onsi/gomega/Dockerfile

```diff
@@ -1 +1 @@
-FROM golang:1.15
+FROM golang:1.16
```

Comment:
One shouldn't update files from the vendor directory, this vulnerability doesn't apply to us, but we'll make it a point to check the use of `golang:1.15` containers in the OpenEBS organization.

NOTE: We should avoid check-ing in the `vendor` as we have moved to Go Mod.